files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -260,6 +260,22 @@
     #
     - variant is regex("external-.*")
 
+- blockinfile:
```

Comment:
FWIW.. I think I'd prefer we deliver a file in `/etc/profile.d`, rather than using ansible to modify `/etc/profile`.. we could deliver that file via Ansible, but even better would be to deliver that file via dpkg (e.g. add it as a file delivered by our package)..

I'm not necessarily opposed to this approach, just wanted to mention perhaps a better way to do this.

Reply:
I chose this way to avoid having a chance of the setting being overridden by other config files. Making it a file in `/etc/profile.d` would introduce a config file load order than would have to be maintained. I might might switch to using a separate config file  if we want to make it configurable in each shell session. Because the purpose of the setting comes from increasing security and passing CIS scans, making it configurable probably is not desirable though. I can foresee reasons for making it configurable but not for any security purposes.

Reply:
OK, sounds good.

Reply:
All of that said, I'm fine either way (and already approved what you have). There are pros and cons either way.

Reply:
having a file in `.d` for our config, also makes it easier for us to change that configuration.. e.g. the logic as written here, only adds the lines.. so, if we wanted to modify the configuration, we'd have to write new logic to remove the existing lines, and then use this logic to add the new lines.. doing this in Ansible becomes tedious, and easy to get wrong.. but only matters once/if we need to change it..

further, w.r.t. the ordering of `.d` files.. that ordering still exists with the `/etc/profile` file itself.. we "solved" this ordering by simply appending to the end of the `/etc/profile` file, but (IMO, at least) we could do something similar by adding a file named `99-delphix`.. i.e. the "99" prefix helps ensure our config is applied "last"..

generally, I think we prefer using `.d` files, as we've done for other configuration files.. e.g.
```
> find . -name '*.d' | head
./files/azure/etc/cloud/cloud.cfg.d
./files/oci/etc/cloud/cloud.cfg.d
./files/common/usr/lib/sysctl.d
./files/common/etc/sudoers.d
./files/common/etc/update-motd.d
./files/common/etc/default/grub.d
./files/common/etc/cloud/cloud.cfg.d
./files/common/lib/udev/rules.d
./files/common/lib/sysctl.d
./files/common/lib/modprobe.d
```

but I don't feel strongly.. and I'm good with this as-is.